From 1bb0ec34c0a4f290392da5484b70ef12fd047e14 Mon Sep 17 00:00:00 2001
From: Ivo Murrell <ivo.murrell@moixa.com>
Date: Wed, 29 Apr 2020 15:10:56 +0100
Subject: [PATCH] Disable roaming.

---
 include/autoconf.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/autoconf.h b/include/autoconf.h
index f5bc920..8cb5e09 100644
--- a/include/autoconf.h
+++ b/include/autoconf.h
@@ -97,8 +97,8 @@
 
 #endif /* CONFIG_CONCURRENT_MODE */
 
-#define CONFIG_LAYER2_ROAMING
-#define CONFIG_LAYER2_ROAMING_RESUME
+// #define CONFIG_LAYER2_ROAMING
+// #define CONFIG_LAYER2_ROAMING_RESUME
 
 #define CONFIG_80211D
 
-- 
2.26.1

