From 301ebaba974278c69b22fff3a68a296da43e8f6c Mon Sep 17 00:00:00 2001
From: Ivo Murrell <ivo.murrell@moixa.com>
Date: Tue, 28 Apr 2020 15:49:50 +0100
Subject: [PATCH] fix makefile for openembedded

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index f5a62f6..7d81916 100644
--- a/Makefile
+++ b/Makefile
@@ -230,8 +230,8 @@ ifeq ($(CONFIG_RASPBIAN), y)
 	EXTRA_CFLAGS += -DCONFIG_P2P_IPS
 ARCH := arm
 CROSS_COMPILE := arm-linux-gnueabihf-
-KVER := $(shell uname -r)
-KSRC ?= /lib/modules/$(KVER)/build
+KVER  := $(KERNEL_VERSION)
+KSRC := $(KERNEL_PATH)
 MODULE_NAME := 8723bu
 MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
 endif
-- 
2.26.1

